### ISSUE TITLE
Closure Compiler Warnings

### ISSUE THREAD

**Issue by geographika** [terrestris/BasiGX#359]
In ExtJS 6.2 the following warnings are shown when building BasiGX:

```
[WRN] C1005: Closure Compiler Warning (Unnecessary escape: '\{' is equivalent to just '{') -- lib\BasiGX\src\plugin\Hover.js:51:46
[WRN] C1005: Closure Compiler Warning (Unnecessary escape: '\{' is equivalent to just '{') -- lib\BasiGX\src\plugin\Hover.js:51:48
[WRN] C1005: Closure Compiler Warning (Unnecessary escape: '\}' is equivalent to just '}') -- lib\BasiGX\src\plugin\Hover.js:59:46
[WRN] C1005: Closure Compiler Warning (Unnecessary escape: '\}' is equivalent to just '}') -- lib\BasiGX\src\plugin\Hover.js:59:48
```

**Comment by marcjansen** [comment]
Thanks for the report, we are using the plugin in several projects and are a bit afraid of simply changing as suggested. We'll need to double check

**Comment by marcjansen** [comment]
Fixed with #363
